schemas/3.0/language.xsd

```diff
@@ -57,6 +57,7 @@
 	  <xs:enumeration value="Turkish" />
 	  <xs:enumeration value="Vietnamese" />	  
 	  <xs:enumeration value="Georgian" />
+      <xs:enumeration value="Catalan" />
```

Comment:
Respect indentation

Reply:
I am aligning with the previous line as the the lines befor 'Catalan' is not.
![image](https://user-images.githubusercontent.com/49319959/91040315-4af93c80-e60e-11ea-904b-2197cbd4d413.png)